scripts/main/album.js

```diff
@@ -1317,19 +1317,58 @@ album.updatePhoto = function (data) {
 			if (this.id === data.id) {
 				this.width = data.width;
 				this.height = data.height;
-				this.small = data.small;
-				this.small_dim = data.small_dim;
-				this.small2x = data.small2x;
-				this.small2x_dim = data.small2x_dim;
-				this.medium = data.medium;
-				this.medium_dim = data.medium_dim;
-				this.medium2x = data.medium2x;
-				this.medium2x_dim = data.medium2x_dim;
-				this.thumbUrl = data.thumbUrl;
-				this.thumb2x = data.thumb2x;
 				this.url = data.url;
 				this.size = data.size;
-
+				// Deep copy size variants
+				this.sizeVariants = {};
+				if(data.sizeVariants.thumb !== null) {
```

Comment:
double check before executing this code that `data.sizeVariants` is not `null`.

Reply:
Fixed with commit 119e753